x/wasm/client/cli/query.go

```diff
@@ -252,20 +252,17 @@ func GetCmdQueryCodeInfo() *cobra.Command {
 			}
 
 			queryClient := types.NewQueryClient(clientCtx)
-			res, err := queryClient.Code(
+			res, err := queryClient.CodeInfo(
```

Comment:
Oh nice, this makes this query much more efficient. No more downloading and discarding the Wasm blob